funlet-whisper/twimlet-whisper.log

```diff
@@ -0,0 +1,42 @@
+[WHISPER-1-1] Recorded Message
```

Comment:
Do we need these logs? Trying to keep the templates as lightweight as possible :)

Reply:
I would say that they are part of the test suite. Should we move all the test files to a different subfolder to clarify?

They allow to compare the [output of the Funlet](https://github.com/twilio-labs/function-templates/issues/17#issuecomment-523669614) with the output of the corresponding Twimlet:

```
$ ./test-call-me.sh "https://$DOMAIN.twil.io/callme" | diff twimlet-call-me.log  -
(...)
$ ./test-whisper.sh "https://$DOMAIN.twil.io/callme" | diff twimlet-whisper.log  - 
(...)
```

Reply:
I have moved them to the `tests` subfolder.